flag/service/kubernetes/kubernetes.go

```diff
@@ -8,9 +8,10 @@ import (
 // Kubernetes is a data structure to hold Kubernetes specific command line
 // configuration flags.
 type Kubernetes struct {
-	Address    string
-	InCluster  string
-	KubeConfig string
-	TLS        tls.TLS
-	Watch      watch.Watch
+	Address        string
+	InCluster      string
+	KubeConfig     string
+	KubeConfigPath string
```

Comment:
It might be clearer to call this `KubeConfig` and call the other `KubeConfigData`. In most Kubernetes applications (thinking of `kubelet`), `--kubeconfig` is the path to the file rather than the file content.

Reply:
Makes sense.

Reply:
...but I think there could be a bit too much disturbance at this point when many places accept the variable but fail during runtime when the string contains filename instead of kubeconfig. Those need to be mapped and then updated in one go. Sounds kind of SWH task for me. I'll got with this for now.

Reply:
s/I'll got with this for now./I'd like to go with this for now./ :slightly_smiling_face:

Reply:
That's true. Just thought I'd mention for the future.